CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-08-13
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with Azure AD Integration (legacy). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to AKS-managed Azure AD automatically starting from 1st Dec. 2023. We recommend to update your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes
+* Features
+  * [Azure Container Networking Interface (CNI) Overlay](https://learn.microsoft.com/azure/aks/azure-cni-overlay) now fully supports Windows Server 2019 and 2022.
+  
+* Behavioral changes
+  * Container Insights Logs Addon (e.g. Monitoring addon) property is migrated from `AddonProfiles` to `Cluster profile`.
+  * Azure monitor metrics addon image is reverted from 07-28-2023 release back to the [06-26-2023 release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-06-26-2023)
+  * Through an annotation, the creation of a [Private Link Service](https://cloud-provider-azure.sigs.k8s.io/topics/pls-integration/) automates to the Azure LoadBalancer, requiring users to only create Private Endpoint connections for private connectivity.
+
+* Component Updates
+  * AKS Image cleaner eraser image bumped to [v1.2.0](https://github.com/eraser-dev/eraser/releases/tag/v1.2.0)
```

Comment:
A couple sentences under component updates are missing a period, can you add them please?